### ISSUE TITLE
NotImplementedError: Unsupported event of type wiki_page

### ISSUE THREAD

**Issue by qvazzler** [NotSqrt/mattermost-integration-gitlab#19]
Any chance of seeing this implemented? Thanks!

**Comment by qvazzler** [comment]
```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/mattermost_gitlab/server.py", line 42, in new_event
    event = event_formatter.as_event(request.json)
  File "/usr/local/lib/python2.7/dist-packages/mattermost_gitlab/event_formatter.py", line 262, in as_event
    raise NotImplementedError('Unsupported event of type %s' % data['object_kind'])
NotImplementedError: Unsupported event of type wiki_page
127.0.0.1 - - [01/Nov/2016 15:54:51] "POST /new_event HTTP/1.1" 200 -

```

**Comment by NotSqrt** [comment]
Shouldn't be too difficult.

It's hard to follow the gitlab releases, I don't have much time to work on this project !

**Comment by qvazzler** [comment]
@NotSqrt Ok. Actually the code pushes don't really work either, but that may be misconfiguration on my part.

**Edit:** Scratch that, I need to run this with `--push` as you mentioned in issue #10
